src/web/wms/src/main/java/org/geoserver/wms/web/publish/WMSLayerConfig.html

```diff
@@ -49,6 +49,27 @@ <h3><wicket:message key="defaultTitle">WMS Settings</wicket:message></h3>
             <select id="defaultInterpolationMethod" wicket:id="defaultInterpolationMethod" class="text"/>
           </li>
         </ul>
+        <ul id="remotestyles" wicket:id="remotestyles">
+        <h3><wicket:message key="remoteDefaultTitle">Cascaded WMS Settings</wicket:message></h3>
+          <li>
+            <label for="remoteStylesDropDown"><wicket:message key="remoteStylesLabel">Select Default Remote Style</wicket:message></label>
+            <select id="remoteStylesDropDown" wicket:id="remoteStylesDropDown" class="text"/>
+          </li>
+          <li>
+            <label for="extraRemoteStyles">Select Additional Remote Styles</label>
```

Comment:
The internationalization is missing, i.e. ``<wicket:message key="``.

Reply:
understood will add that

Reply:
kk